tests/features/random_trial_test.py

```diff
@@ -37,8 +107,10 @@ def test_initialize_random_trial_pca_bridge(make_model, params):
         if bridges:
             bridge_num += 1
             for agent in comp.nodes():
-                if agent.pca.active:
-                    assert agent in [ag for ags in bridges for ag in ags]
+                if agent.knowledge.active:
+                    assert agent in [
+                        ag for ags in bridges for ag in ags
+                    ]  # TO_REVIEW what?
```

Comment:
I think what this is testing is that the chosen agents are actually in the bridging agent list it should be choosing from?